cli.php

```diff
@@ -34,8 +34,17 @@
 
 // Create the CLI command instance and add to WP CLI.
 $plugin_command = new Plugin_Check_Command( $context );
-WP_CLI::add_command( 'plugin', $plugin_command );
-
+WP_CLI::add_command(
+	'plugin',
+	$plugin_command,
+	array(
+		'after_invoke' => function () {
+			if ( file_exists( ABSPATH . 'wp-content/object-cache.php' ) ) {
+				unlink( ABSPATH . 'wp-content/object-cache.php' );
+			}
```

Comment:
That sounds too easy to be true 😅

Reply:
Yah, looks like it does not work :-D

Reply:
![Screenshot 2024-08-20 at 3 21 39 PM](https://github.com/user-attachments/assets/c8cb7235-7808-404a-9ba7-f381c1fabec9)

Reply:
But tests are passing in GH Action.

Reply:
Can you point me to where in the Behat tests the failing call is defined? I'm not familiar with Behat _at all_, so not sure whether I'm looking at the right place. Is it the _second_ `plugin check` execution (in https://github.com/WordPress/plugin-check/pull/518/files#diff-303aba0b6bdf4d673c51a01f6bb7b91eccfc4717908349148e02927f9f3ac223R602) specifically that's failing? Or is even the first `plugin check` failing?

Reply:
Yes. First command run good. When same command run again, issue appears.

Reply:
Thanks. Can you also clarify how that relates to runtime checks? As far as I can tell, the addon checks are all non-runtime checks, so what does that have to do with runtime checks / setting up the `object-cache.php` drop-in?

Reply:
If that file is deleted manually then above issue does not appear. This is not specific to runtime checks.

Reply:
But isn't that file only created _after_ the failing second run, in https://github.com/WordPress/plugin-check/pull/518/files#diff-303aba0b6bdf4d673c51a01f6bb7b91eccfc4717908349148e02927f9f3ac223R672? Since only there the `--require` is used, so only that should place the drop-in. 🤔